php/media/class-gallery.php

```diff
@@ -236,15 +243,6 @@ public function block_editor_scripts_styles() {
 			$this->media->plugin->version,
 			true
 		);
-
-		wp_localize_script(
```

Comment:
Is this no longer necessary?

Reply:
Opted for using WP's `apiFetch`, which handles this part out of the box.